docs: update the azure video (#225)
livecycle/preevy: site/docs/drivers/azure.md

Preevy uses the Microsoft Azure SDK which can obtain the application [default credentials](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/identity/identity#defaultazurecredential).
The simplest way is to use `az login` command.

> Also, you can check out the video below for a step-by-step guide on how to configure Azure credentials ans use them with Preevy.

Reviewer: ```suggestion
See the video below for a step-by-step guide on how to configure Azure credentials ans use them with Preevy.
```
